### DISCUSSION TITLE
Is there a way I could interpolate my data

### DISCUSSION THREAD

**Discussion by delahoussou** [pydata/xarray#7998] [Q&A]
I have this data that I want to interpolate and get a higher resolution for lat and lon. I want to get my data to be (1200, 1600)  in shape. As at now raw_data is (11,15)

    path = "C:\\Users\\dhoussou\\Documents\\CH2018_pr_SMHI-RCA_GFDL_EUR44_RCP45_QMgrid_1981-2099.nc"
    data = xr.open_dataarray(path)
    
    lat_extent = [46.759778375, 46.926506988]
    lon_extent = [9.287051783, 9.629031007]
    lats = data["lat"][:]
    lons = data["lon"][:]
    times = data["time"][:]
    total_steps = len(times)
    
    lat_min = (np.abs(lats - lat_extent[0])).argmin()
    lat_max = (abs(lats- lat_extent[1])).argmin()
    lon_min = (abs(lons- lon_extent[0])).argmin()
    lon_max = (abs(lons- lon_extent[1])).argmin()
    
    raw_data = data.isel(lon = np.arange(161,179,1), lat =np.arange(47,56,1))
    
    yearly_precip = raw_data.groupby("time.year")
    Average = yearly_precip.mean()

**Comment by TomNicholas** [comment]
Hi @DOptimusPrime - have you read our [documentation page on interpolating data](https://docs.xarray.dev/en/stable/user-guide/interpolation.html)?
